mcstatus/querier.py

```diff
@@ -132,6 +132,8 @@ def __init__(self, version: str, plugins: str):
     map: str
     players: Players
     software: Software
+    hostip: str
+    hostport: int
```

Comment:
It is provided in `server.address`, no need to duplicate it

Reply:
There's a potential difference, right? One being from the client's request and the other (here) being from the server's status response. Does this matter?